proxy/src/control/discovery.rs

```diff
@@ -19,6 +17,8 @@ use conduit_proxy_controller_grpc::destination::Update as PbUpdate;
 use conduit_proxy_controller_grpc::destination::update::Update as PbUpdate2;
 use conduit_proxy_controller_grpc::destination::client::{Destination as DestinationSvc};
 
+use control::cache::{Cache, CacheChange, Exists};
```

Comment:
TIOLI: I wonder if the `Cache` module should just be namespaced under `discovery` instead? Do we expect this code to be used elsewhere?

Reply:
there's nothing specific to discovery in this module, and i anticipate that other streaming endpoints will need the same sort of logic.